tools/docker-receptor.yml

```diff
@@ -17,7 +17,7 @@ services:
     volumes:
       - "./docker-compose-cluster:/etc/receptor"
   receptor-hop:
-    image: quay.io/ansible/awx-ee:devel
+    image: quay.io/project-receptor/receptor:latest
```

Comment:
@fosterseth you should find this interesting.

The jump nodes _really don't need anything_ other than receptor. It's the academically correct choice, then, to use the pure receptor image. This gets us much closer to what we're going for. You can't run jobs on this node, or use podman inside of it.

And most importantly, it actually works too.

Reply:
perfection

Reply:
ditto, this makes sense